scrapers/AnalizedNetwork.yml

```diff
@@ -0,0 +1,39 @@
+name: "Analized"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - analbbc.com/1/scene/
+      - analviolation.com/1/scene/
+      - analized.com/1/scene/
+      - baddaddypov.com/1/scene/
+      - dtfsluts.com/1/scene/
+      - girlfaction.com/1/scene/
+      - hergape.com/1/scene/
+      - homemadeanalwhores.com/1/scene/
+      - jamesdeen.com/1/scene/
+      - onlyprince.com/1/scene/
+      - pervertgallery.com/1/scene/
+      - povperverts.net/1/scene/
+      - teenageanalsluts.com/1/scene/
+      - twistedvisual.com/1/scene/
+      - yourmomdoesanal.com/1/scene/
+      - yourmomdoesporn.com/1/scene/
+      - downtofuckdating.com/1/scene/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="grid-x"]//h4[@class="text-center"]
+      Details: //p[@class="hide-for-small-only"]/text()
```

Comment:
the `text()` should be removed as it cuts the description on the first span
```yaml
Details: //p[@class="hide-for-small-only"]
```
is enough since stash can get the innertext from the paragraph